alica_test_utility/src/Util.cpp

```diff
@@ -8,16 +8,27 @@ namespace alica::test
 {
 BasicBehaviour* Util::getBasicBehaviour(alica::AlicaEngine* ae, int64_t behaviourID, int64_t configurationID)
 {
-    BasicBehaviour* behaviour = nullptr;
-    for (auto& behaviourEntry : ae->getBehaviourPool().getAvailableBehaviours()) {
-        if (behaviourEntry.first->getAbstractPlan()->getId() == behaviourID &&
-                (configurationID == 0 ? behaviourEntry.first->getConfiguration() == nullptr
-                                      : behaviourEntry.first->getConfiguration()->getId() == configurationID)) {
-            behaviour = behaviourEntry.second.get();
-            break;
+    return getBasicBehaviourHelper(ae->getPlanBase().getRootNode(), behaviourID);
```

Comment:
I guess later we should refactor these to only take the part of the engine we need

Reply:
Could be in a separate pr, configuration is also unused.